.github/actions/verify-token/src/predicate.ts

```diff
@@ -118,10 +113,21 @@ export interface githubObj {
   actor: string;
 }
 
+function getWorkflowPath(repoName: string, workflowRef: string): string {
+  // GITHUB_WORKFlOW_REF contains the repository name in the path. We will trim
```

Comment:
Can you provide an example of `GITHUB_WORKFlOW_REF` content in the comment?

nit: GITHUB_WORKFlOW_REF -> GITHUB_WORKFLOW_REF

Reply:
Done.